hwdb.d/60-keyboard.hwdb

```diff
@@ -1099,13 +1080,6 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnIBM*:pn18666TU:*
  KEYBOARD_KEY_6b=esc                                    # escape
  KEYBOARD_KEY_69=enter                                  # enter on d-pad
 
-# IdeaPad
-evdev:name:Ideapad extra buttons:dmi:bvn*:bvr*:bd*:svnLENOVO*:pn*:*
- KEYBOARD_KEY_0d=rfkill                                 # airplane mode switch (toggle all wireless devices)
```

Comment:
I'm afraid I made a mistake here in my previous review, the kernel maps 0d -> wlan not rfkill. For GNOME this does not make a difference but it does e.g. make a difference for the in kernel rfkill handling which is used when userspace has not claimed rfkill control.

So if we want to avoid any behavior changes the "KEYBOARD_KEY_0d=rfkill" needs to be preserved, sorry about this.